modules/images-configuration-registry-mirror-convert.adoc

```diff
@@ -12,6 +12,8 @@ Using an `ImageContentSourcePolicy` (ICSP) object to configure repository mirror
 
 ICSP objects are being replaced by `ImageDigestMirrorSet` and `ImageTagMirrorSet` objects to configure repository mirroring. If you have existing YAML files that you used to create `ImageContentSourcePolicy` objects, you can use the `oc adm migrate icsp` command to convert those files to an `ImageDigestMirrorSet` YAML file. The command updates the API to the current version, changes the `kind` value to `ImageDigestMirrorSet`, and changes `spec.repositoryDigestMirrors` to `spec.imageDigestMirrors`. The rest of the file is not changed.
 
+Because the migration results in the same `registries.conf` file, the cluster does not need to reboot. 
```

Comment:
@QiWang19 Would changing this line to the following be accurate?

> Because the migration does not change the `registries.conf` file, the cluster does not need to reboot.

Reply:
yes, that's correct.